addons/mail/static/src/messaging/component/message_list/message_list.js

```diff
@@ -21,8 +21,9 @@ class MessageList extends Component {
             const threadViewer = this.env.entities.ThreadViewer.get(props.threadViewerLocalId);
             const thread = threadViewer ? threadViewer.thread : undefined;
             const threadCache = threadViewer ? threadViewer.threadCache : undefined;
+            const isDeviceMobile = this.env.messaging && this.env.messaging.device.isMobile;
```

Comment:
this guard should no longer be necessary when https://github.com/odoo-dev/odoo/pull/230 is merged (very soon)

Reply:
Will remove when #230 is merged, just keeping it to have green runbot